docs/explanations/pro_token_and_machine_usage.rst

```diff
@@ -0,0 +1,49 @@
+.. _pro_token_and_machine_usage:
+
+Machine counts and token usage for Ubuntu Pro
+*********************************************
+
+Ubuntu Pro tokens are used to authenticate your machine and enable access to the services you are entitled to. These tokens are unique and specific to your account, so you should treat them as a secret (as you would with a password).
+
+In this page, we'll discuss how Ubuntu Pro tokens work and how machines are counted against your subscription.
+
+Ubuntu Pro token(s)
+--------------------
+
+- Obtaining Pro tokens:
+     After subscribing to Ubuntu Pro, you can access your subscriptions through the `Ubuntu Pro Dashboard <Pro_dashboard_>`_. Each subscription includes a Ubuntu Pro token that can be used to attach as many machines as the subscription allows.
+
+- Attaching Pro tokens:
+    Command: Use ``sudo pro attach`` to connect a machine to your Ubuntu Pro subscription.
+
+- Detaching Pro tokens:
+    Command: Use ``sudo pro detach`` to remove a machine from the subscription. If you are attached on a VM you intend to delete, the token does not need to be detached before the VM is destroyed.
+
+For further information regarding pro commands, refer to our :ref:`Getting started-with Ubuntu Pro client tutorial<tutorial-commands>`.
+
+Monitoring active machines:
+---------------------------
+
+Active machines are counted based on the number of machines attached to your subscription at a given time. An active machine is defined as a machine that has been attached to a given subscription at any point within the past 24 hours.
+
+Each subscription type has a limit on the number of machines it can cover. For personal subscriptions, this limit is 5 physical machines with unlimited VMs on these machines. Enterprise subscriptions will have a higher limit and may differentiate between physical and virtual machines.
+For use beyond 5 physical machines, consider an enterprise subscription.
+
+The Ubuntu Pro dashboard can help track the number of active machines, ensuring compliance with subscription limits. It provides a comprehensive view of all active and attached machines. It helps track the number of active machines, ensuring that you remain within the subscription limits.
+
+Dealing with multiple VMs:
+---------------------------
+
+In an IaaS environment where node control is limited, you can contact `Canonical's sales team <pro_sales_form_>`_ to discuss purchasing licenses per single VM rather than per physical node.
```

Comment:
What's the advantage to buying a license for a single VM? How do VMs and nodes relate to each other? I.e., if you have a license for a node, how many VMs might that include and therefore what's the advantage to buying an individual license (not sure if this info is in the scope of this document, but it's what I'm left wondering as a reader)

Reply:
I think some of these questions make sense but they might be more specific in nature, I would assume if there was a seperate document which explained VM licensing or even a sales/customer success document explaining these differences, it would be better?

Reply:
I haven't seen anything like that, except what Lech and Rajan have mentioned in the FAQ replies :/ I'm not sure if that even exists

Reply:
Yeah the issue is, this information might be too specific and the FAQ's don't really talk about these in detail. @orndorffgrant any suggestions for this?

Reply:
I think since we don't know of any good resources or references, we can note it down for future work to include it here. It's not crucial enough to hold up the PR over it.